### PR TITLE
Update sites.yml

### PR DESCRIPTION
**Pull request by blmayer**
Updated blmayer.dev
Added saucecode.bar

This PR is:

- [x] Adding a new domain
- [x] Updating existing domain **size**
- [x] Changing domain name
- [ ] Removing existing domain from list
- [ ] Website code changes (512kb.club site)
- [ ] Other not listed

- [ ] I used the uncompressed size of the site
- [x] I have included a link to the GTMetrix report
- [x] The domain is in the correct alphabetical order
- [ ] This site is not a ultra lightweight site
- [ ] The following information is filled identical to the data file

***I confirm that I have read the [FAQ section](https://512kb.club/faq), particularly the two red items around minimal pages and inappropriate content and I attest that this site is neither of these things.***

- [x] Check to confirm

GTMetrix Report: https://gtmetrix.com/reports/saucecode.bar/hI7TQgn8/

### CODE REVIEW

File: _data/sites.yml [hunk 1]
```diff
@@ -304,9 +304,9 @@
   last_checked: 2022-05-15
 
 - domain: blmayer.dev
-  url: https://blmayer.dev/
-  size: 20.4
-  last_checked: 2022-03-25
+  url: https://www.blmayer.dev/
+  size: 5.5
```
Comment (garritfra) [inline _data/sites.yml:308]: Could you provide a GTMetrix link to verify this?

File: _data/sites.yml [hunk 2]
```diff
@@ -1993,6 +1993,11 @@
   size: 91.1
   last_checked: 2022-03-21
 
+- domain: saucecode.bar
+  url: https://saucecode.bar/
+  size: 5.6
```
Comment (garritfra) [inline _data/sites.yml:1998]: Nitpick: this should be 5.61, but I'll let it slide. ;)

**Review by garritfra — CHANGES_REQUESTED** [review]

File: _data/sites.yml (hunk 2 above)
Comment (blmayer) [reply _data/sites.yml:1998]: Oops, Thanks.

File: _data/sites.yml (hunk 1 above)
Comment (blmayer) [reply _data/sites.yml:308]: https://gtmetrix.com/reports/www.blmayer.dev/boDvNHqJ/

Comment (garritfra) [reply _data/sites.yml:308]: According to that, the site is 7.53kb. could you update that?

Comment (blmayer) [reply _data/sites.yml]: Yes, I updated.

File: _data/sites.yml (hunk 2 above)
Comment (blmayer) [reply _data/sites.yml]: I updated this size as well.

**Review by garritfra — APPROVED** [review]

**Pull request merged**